types/styled-system/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "styled-components": "^3.3.2"
+    }
```

Comment:
Does ‘private’ mean the dependency won’t be included in the published package? (As this appears to only be for the tests)

Reply:
I think that's the case, yes.

Reply:
Seems to be a runtime dependency, so this isn’t good https://www.npmjs.com/package/@types/styled-system

Reply:
we looked at other projects, and they never used devDeps here - so not too sure what to do with this

Reply:
Then we should probably just build a dummy `styled` function in these tests that has the same typing.